simple/rdm_inject_pingpong.c

```diff
@@ -479,92 +469,51 @@ static int run(void)
 	return ret;
 }
 
-void print_usage(char *name)
-{
-	fprintf(stderr, "Usage:\n");
-	fprintf(stderr, "  %s [OPTIONS]\t\tstart ping server\n", name);
-	fprintf(stderr, "  %s [OPTIONS] <host>\tpong given host\n", name);
-
-	fprintf(stderr, "\nOptions:\n");
-	fprintf(stderr, "  -n\tdomain name\n");
-	fprintf(stderr, "  -p\tport number\n");
-	fprintf(stderr, "  -s\tsource address\n");
-	fprintf(stderr, "  -I\tnumber of iterations\n");
-	fprintf(stderr, "  -S\tspecific transfer size or 'all'\n");
-	fprintf(stderr, "  -m\tmachine readable output\n");
-	fprintf(stderr, "  -i\tprint hints structure and exit\n");
-	fprintf(stderr, "  -v\tdisplay versions and exit\n");
-	fprintf(stderr, "  -h\tdisplay this help output\n");
-
-	return;
-}
-
 int main(int argc, char **argv)
 {
 	int op, ret;
-	int prhints = 0;
 
-	while ((op = getopt(argc, argv, "n:p:s:I:S:mivh")) != -1) {
+	/* default options for test */
+	opts.iterations = 1000;
+	opts.transfer_size = 1024;
+	opts.port = "9228";
+	opts.argc = argc;
+	opts.argv = argv;
```

Comment:
I wonder if we shouldn't create an INIT_OPTS macro to set a few of these values by default, especially the argc/argv values.  Just to make it less likely someone will forget to set one.